docs/3.0.0-alpha.x/guides/graphql.md

```diff
@@ -456,13 +456,13 @@ module.exports = {
 };
 ```
 
-- `definition` (string): let's you define new type, input, etc.
+- `definition` (string): let you define new type, input, etc.
```

Comment:
You are correct that "let's" is not correct here.  But it should be "lets" <--- please see https://english.stackexchange.com/questions/3392/lets-vs-lets-which-is-correct